src/utils/barcode-scan.js

```diff
@@ -13,6 +21,16 @@ const readImageFile = async (file) => {
   return returnedImage;
 };
 
+/*
+ * Decodes the barcode using @zxing library,
+ * returning the data as text
+ *
+ * @memberof utils
+ * @function decodeBarcode
+ * @param {object} image - raw returned image data from readImageFile
+ * @returns {string} result.text - decoded text data barcode image
```

Comment:
Because this is an async function, it always returns a Promise. However, I think you can type the thing that the promise returns, like Promise<string>. It's a bit like TypeScript. See example below:

<img width="559" alt="Screen Shot 2023-05-24 at 4 28 20" src="https://github.com/codeforpdx/PASS/assets/14917816/44396289-1b96-4f9d-a3b8-95a67f108f08">

<img width="417" alt="Screen Shot 2023-05-24 at 4 28 35" src="https://github.com/codeforpdx/PASS/assets/14917816/df72c921-6dd1-4017-a08b-bef5e1857881">